src/frontend/components/chat-bottom-bar.tsx

```diff
@@ -26,12 +26,25 @@ const ChatBottomBar = ({ onSend }: Props) => {
 			id: '-1'
 		}
 	);
+	const [buttonClass, setButtonClass] = useState(
+		'rounded-full flex-shrink-0 transition-opacity duration-200 opacity-50'
+	);
 	const { isMobile } = useWindowSize();
 
 	useEffect(() => {
 		setIsMounted(true);
 	}, [isMounted]);
 
+	useEffect(() => {
+		setButtonClass(
+			`rounded-full flex-shrink-0 transition-opacity duration-200 ${
+				message.content.trim()
+					? 'opacity-100'
+					: 'opacity-50'
+			}`
+		);
+	}, [message.content]);
```

Comment:
You could instead use a disabled variable. Makes it easier when you extend it with global state.